test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -483,7 +484,28 @@ def setUp(self):
         # once save & load configurations are implemented for
         # comm, mom, scheduler
         if self.use_cur_setup:
-            self.server.delete_nodes()
+            self.server.delete_nodes(True)
+            attr = {}
+            test_user = pwd.getpwuid(os.getuid())[0] + '@*'
```

Comment:
Now when you are using sudo everywhere why do we need to set mgrs and opers?

Reply:
The `revert_moms` is failing. Also, In the test case, the user may create vnodes, queues or do some other manager oriented operations. So I feel adding them will be the right thing to do.

Reply:
"Also, In the test case, the user may create vnodes, queues or do some other manager oriented operations"
This test case has to take care of it, not setUp.

"The revert_moms is failing."
I suggest using sudo there also... We should keep one way of doing stuff, either sudo or with mgrs/opers... not both of it.